lib/src/image_inlining/image_dart_compiling.dart

```diff
@@ -0,0 +1,39 @@
+import 'dart:async';
+import 'image_inliner.dart';
+import 'package:path/path.dart';
+
+/// Data holder for image-related information: its file name and contents.
+class ImageInformation {
+  final String fileName;
+  final List<int> contents;
+
+  ImageInformation(this.fileName, this.contents);
+}
+
+/// Convert file name to a string suitable for being used as Dart identifier.
+/// Currently result isn't guaranteed to be always valid identifier, but
```

Comment:
Can we go the extra mile of guaranteeing it?

return fileName.replaceAll
name.replaceAll(new RegExp(r'[^\w]+'), '_').replaceAll(new RegExp(r'^(\d|_)'), ...prepend some char to make sure we don't start with an _ or a digit))